### ISSUE TITLE
[chrome-launcher] Memory leak: cleanup SIGINT when instance is killed

### ISSUE THREAD

**Issue by andrejewski** [GoogleChrome/lighthouse#2797]
Every time a new Chrome instance is launched via `chrome-launcher`'s [`.launch()`](https://github.com/GoogleChrome/lighthouse/blob/07817f90e538dbbd2e8e430e12a1deab569d718c/chrome-launcher/chrome-launcher.ts#L56) a listener is placed on the SIGINT event which is never cleaned up. After 10 launches, Node warns about a potential memory leak:

```
(node:23024) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added. Use emitter.setMaxListeners() to increase limit
```

The [kill function returned from `launch()`](https://github.com/GoogleChrome/lighthouse/blob/07817f90e538dbbd2e8e430e12a1deab569d718c/chrome-launcher/chrome-launcher.ts#L64) should clean this listener up.

**Comment by mikecardwell** [comment]
I've provided an alternative fix which I believe is better. The one provided by JarrodCTaylor would still trigger the EventEmitter memory leak warnings for me, as I'm starting up lots of Chrome instances at the same time. My fix ensures that there is a maximum of one SIGINT listener at any one point in time.

**Comment by paulirish** [comment]
fixed by #2959
